src/main/java/Duke/Event.java

```diff
@@ -0,0 +1,37 @@
+package Duke;
+
+/**
+ * Creates an Event task.
+ */
+public class Event extends Task {
+    protected String at;
+
+    /**
+     * Constructor
+     *
+     * @param description name of task
+     */
+    public Event(String description, String at) {
+        super(description);
+        this.at = at;
```

Comment:
Perhaps consider using a variable name that is able to share more details about the data it holds to make it easier to understand its purpose